lib/middleware/websocket/interactors/handle_message.rb

```diff
@@ -23,6 +36,12 @@ def update_player_position(data:, room_id:)
           data: data, room_id: room_id
         )
       end
+
+      def player_verified?(uuid:)
+        player = Interactors::Players::FetchPlayer.new.call(uuid: uuid).player
+
+        player.nil? ? false : true
```

Comment:
This can just be `player.nil?`. Since Ruby methods evaluate to their last expression it would return `true` or `false`.